astropy/nddata/utils.py

```diff
@@ -27,9 +35,18 @@ def overlap_slices(large_array_shape, small_array_shape, position):
         Shape of the large array.
     small_array_shape : tuple
         Shape of the small array.
-    position : tuple
+    position : tuple of integers
         Position of the small array's center, with respect to the large array.
         Coordinates should be in the same order as the array shape.
+        When determining the center for a coordinate with an even number of
+        elements, the position is rounded up. So the coordinates of the
+        center of an array with shape=(2,3) will be (1,1).
+    mode : ['partial', 'strict']
+        In "partial" mode, a partial overlap of the small and the large
+        array is sufficient. In the "strict" mode, the small array has to be
+        fully contained in the large array, otherwise an
+        `IncompleteOverlapError` is raised. In both modes,
+        non-overlapping arrays will raise a `~nddata.utils.NoOverlapError`.
```

Comment:
I think this still needs to be fully spelled out as `~astropy.nddata.util.NoOverlapError`.  You might want to check any other references like it as well.